buildSrc/src/main/java/Config.kt

```diff
@@ -1,14 +1,16 @@
 import java.math.BigDecimal
 
 object Config {
-    val kotlinVersion = "1.5.31"
+    val kotlinVersion = "1.6.10"
```

Comment:
We also need this to support compose latest version. @marandaneto do you see any problems with bumping this for the whole monorepo? Kotlin now is at 1.7.

Reply:
We still compile with `kotlinCompatibleLanguageVersion = "1.4"`.
Is that still possible with Kotlin 1.6, right?
We can consider using `1.5` for the `kotlinCompatibleLanguageVersion`.
Do we know the usage of Kotlin versions?
It's a matter of back-compatibility here.

Reply:
Oh yeah, I think we can still keep it at 1.4 actually, it's still possible with 1.6.

The language adoption is not available publicly unfortunately.

Reply:
will do 👍